src/content/docs/developers/tools.mdx

```diff
@@ -24,25 +24,25 @@ Create a banner that appears when your visitors don't have the Web Monetization
 </script>
 ```
 
-The script automatically web monetizes your site. If your page contains [multiple](/developers/link-element#multiple-monetization-links) monetization links, payments from the extension are only split between unique payment pointers/wallet addresses. 
+The script automatically web monetizes your site. If your page contains [multiple](/developers/link-element#multiple-monetization-links) monetization links, payments from the extension are only split between unique wallet addresses. 
```

Comment:
Extension doesn't do a uniqueness check.

```suggestion
The script automatically web monetizes your site. If your page contains [multiple](/developers/link-element#multiple-monetization-links) monetization links, payments from the extension are only split between wallet addresses. 
```

Reply:
From Arpi:

> payments from the extension are split between the unique addresses present in the page (so link rels with the same address only count as 1 but widget one time payments go toward the wallet address configured for the widget, regardless of how many addresses are in the page added by link rel tag

I just want to make sure we're all on the same page. That comment was from March 6th, so I wouldn't be surprised if something changed 😁

Reply:
> widget one time payments go toward the wallet address configured for the widget, regardless of how many addresses are in the page added by link rel tag

That is the behavior of payment from widget, regardless of presence/absence of extension. Correct.

But the extension (and Chromium implementation) doesn't check if link tags have duplicate wallet addresses. This has never changed.
User can duplicate a link tag 10 times per page, and in that case, we'll send 10 smaller payments split across link tags/wallet addresses.